packages/codemod/src/upgrade-legacy.ts

```diff
@@ -90,8 +90,14 @@ const upgradeLegacy = async () => {
         },
       })
       let createdConfig = config.get().value.right
-
-      addNamedImport(program, "withBlitz", "@blitzjs/next")
+      let importWithBlitz = j.expressionStatement(
+        j.assignmentExpression(
+          "=",
+          j.identifier("const { withBlitz }"),
+          j.callExpression(j.identifier("require"), j.identifier("@blitzjs/next")),
```

Comment:
I just tested this in astexplorer.net

I believe the second argument for `callExpression` should be an array. We also need to include quotes around `@blitzjs/next`.

This worked in astexplorer:

```ts
        let importWithBlitz = j.expressionStatement(
        j.assignmentExpression(
          "=",
          j.identifier("const { withBlitz }"),
          j.callExpression(j.identifier("require"), [j.identifier(`"@blitzjs/next"`)]),
        )
      )
```

Reply:
yeah my bad. Just tested it again.  I will update the code

Reply:
Perfect. Thank you kind sir @siddhsuresh!